controllers/openstackcluster_controller.go

```diff
@@ -365,18 +365,14 @@ func reconcileBastion(log logr.Logger, osProviderClient *gophercloud.ProviderCli
 }
 
 func reconcileNetworkComponents(log logr.Logger, osProviderClient *gophercloud.ProviderClient, clientOpts *clientconfig.ClientOpts, cluster *clusterv1.Cluster, openStackCluster *infrav1.OpenStackCluster) error {
+	var loadBalancerService *loadbalancer.Service
```

Comment:
You can remove the initialization, as it‘s only used once inside the condition :)

Reply:
right, forgot this ..